src/components/logo/logo.tsx

```diff
@@ -64,15 +64,21 @@ class Logo extends Component<any, any> {
     const url = this.props.config.url;
     if (url && url.indexOf(ENTRY_VAR) !== -1) {
       const {player, eventManager} = this.props;
-      eventManager.listen(player, player.Event.MEDIA_LOADED, () => {
-        const entryId = player.sources.id;
-        if (typeof entryId === 'string') {
-          this.setState({urlLink: url.replace(ENTRY_VAR, entryId)});
-        } else {
-          this.props.logger.debug(`Logo url was not replaced; entry id was not found.`);
-          this.setState({isUrlClickable: false});
-        }
-      });
+      const entryId = player.sources.id;
+      if (typeof entryId === 'string') {
+        this.setState({urlLink: url.replace(ENTRY_VAR, entryId)});
+      } else {
+        this.setState({isUrlClickable: false});
```

Comment:
Why it is set to true from the beginning? If it depends on condition
One  you solve this , it will made the initialisation here  unnecessary

Reply:
But you it is still depends on optional url config.. so  by default it should be false.. if I get it right

Reply:
so you're just suggesting that it would be the opposite. starts with false and set to true in specific cases?